transformer_engine/pytorch/module/base.py

```diff
@@ -588,7 +592,6 @@ def prepare_forward(
                 self.fp8
                 and self.training
                 and is_fp8_activation_recompute_enabled()
-                and not in_fp8_activation_recompute_phase()
```

Comment:
Do we expect this change to affect behavior? I can imagine some pathological cases where we do multiple recomputations in the backward pass, e.g. recursive activation checkpointing.

Reply:
Because of the logic here, this condition is always `True`.